hack/e2e/run-rp-and-e2e.sh

```diff
@@ -65,9 +65,31 @@ register_sub() {
 }
 
 run_e2e() {
-    RESOURCEGROUP=$ARO_RESOURCEGROUP make test-e2e
+    export E2E_ENTRYPOINT="docker run --privileged \
+                --network host \
+                --env-file <(env | egrep 'AZURE|E2E|ARO') \
+                --env LOCATION \
+                --env CI \
+                --env RESOURCEGROUP=$ARO_RESOURCEGROUP \
```

Comment:
```suggestion
                --env RESOURCEGROUP \
```
since you are doing RESOURCEGROUP=$ARO_RESOURCEGROUP below